Assets/Scripts/Cards/DisplayCard.cs

```diff
@@ -2,30 +2,33 @@
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.UI;
+using TMPro;
 
 public class DisplayCard : MonoBehaviour {
```

Comment:
Could we call this class something like "CardView" or "CardViewController" instead? In Visual Studio you can refactor a class name by pressing F2 when you've got it selected